src/bandersnatch/main.py

```diff
@@ -15,11 +15,16 @@
 import bandersnatch.verify
 
 from .configuration import BandersnatchConfig
+from .filter import filter_project_plugins, filter_release_plugins
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger(__name__)  # pylint: disable=C0103
```

Comment:
Why did this start firing? I also don't get why it's a `invalid-name` ... You got any insight here?

Reply:
With default pylint configuration it triggers an invalid name because it wants constants to be upper case.